Re enable debian 9 check

disabled in
https://github.com/dotnet/core-sdk/pull/6705

The error is 
```
  Now running lintian...
  E: dotnet-sdk-5.0 source: source-is-missing sdk/5.0.100-ci/AppHostTemplate/apphost
  W: dotnet-sdk-5.0 source: maintainer-script-lacks-debhelper-token debian/postinst
  W: dotnet-sdk-5.0 source: ancient-standards-version 3.9.2 (current is 3.9.8)
  E: dotnet-sdk-5.0: arch-dependent-file-in-usr-share usr/share/dotnet/sdk/5.0.100-ci/AppHostTemplate/apphost
  W: dotnet-sdk-5.0: new-package-should-close-itp-bug
  E: dotnet-sdk-5.0: description-contains-invalid-control-statement
  W: dotnet-sdk-5.0: extended-description-line-too-long
  W: dotnet-sdk-5.0: extra-license-file usr/share/dotnet/sdk/5.0.100-ci/Sdks/Microsoft.NET.Sdk.WindowsDesktop/LICENSE.TXT
  E: dotnet-sdk-5.0: unknown-control-interpreter control/postinst #!/usr/bin/env
  W: dotnet-sdk-5.0-dbgsym: debug-file-with-no-debug-symbols usr/lib/debug/.build-id/07/3309215d63d6cf1ddde7f822d87ff388807b18.debug
  Finished running lintian.
  
  WARNING generated by debuild:
  Making debian/rules executable!
  
/home/vsts/work/1/s/src/redist/targets/GenerateDebs.targets(301,7): error : 2 .deb files generated. [/home/vsts/work/1/s/src/redist/redist.csproj]
##[error]src/redist/targets/GenerateDebs.targets(301,7): error : 2 .deb files generated.

Build FAILED.

/home/vsts/work/1/s/src/redist/targets/GenerateDebs.targets(301,7): error : 2 .deb files generated. [/home/vsts/work/1/s/src/redist/redist.csproj]
    0 Warning(s)
    1 Error(s)
```

--- a/.vsts-ci.yml
+++ b/.vsts-ci.yml
@@ -106,6 +106,12 @@ stages:
               _LinuxPortable: ''
               _RuntimeIdentifier: ''
               _BuildArchitecture: 'x64'
+            Build_Debian_Stretch_Debug_x64:
+              _BuildConfig: Debug
+              _DockerParameter: '--docker debian'
+              _LinuxPortable: ''
+              _RuntimeIdentifier: ''
+              _BuildArchitecture: 'x64'
             Build_Rhel_7_2_Release_x64:
               _BuildConfig: Release
               _DockerParameter: '--docker rhel'

--- a/src/redist/targets/GenerateDebs.targets
+++ b/src/redist/targets/GenerateDebs.targets
@@ -293,9 +293,12 @@
                      PackageVersion="$(Version)"
                      WorkingDirectory="$(DotnetDebToolDir)" />
 
-      <!-- Copy SDK package to output -->
+      <!-- Copy SDK package to output
+          dbgsym packages is Debian Debug Symbol Packages https://wiki.ubuntu.com/Debug%20Symbol%20Packages,
+          we don't use them.
+      -->
       <ItemGroup>
-        <GeneratedDebFiles Include="$(DotNetDebToolOutputDirectory)/*.deb" />
+        <GeneratedDebFiles Include="$(DotNetDebToolOutputDirectory)/*.deb" Exclude="$(DotNetDebToolOutputDirectory)/*dbgsym*.deb" />
       </ItemGroup>
 
       <Error Text="@(GeneratedDebFiles->Count()) .deb files generated." Condition="'@(GeneratedDebFiles->Count())' != 1" />

